src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarViewBaseItem.h.cs

```diff
@@ -24,8 +24,14 @@ public partial class CalendarViewBaseItem : Control
 #endif
 			// Uno only
 			Initialize_CalendarViewBaseItemChrome();
+			this.Loaded += (_, _) =>
+			{
+#if __ANDROID__ || __IOS__ || __SKIA__ || __WASM__ || __MACOS__
```

Comment:
@MartinZikmund not a big deal, but `HAS_UNO` would have been enough?

Reply:
Intent is likely to exclude reference API.

Reply:
Yes, it will be possible to avoid this when I finally merge the `BorderLayerRenderer` refactor